model/src/main/proto/topic.proto

```diff
@@ -249,3 +251,34 @@ message CellularDataPreference {
   // Preference on whether to use cellular data
   bool use_cellular_data = 2;
 }
+
+
+message Subtopic {
+
+  string subtopic_id = 1;
```

Comment:
Add documentation for Protos and its datatypes. Similar to line 268.

Reply:
added.